attributes/default.rb

```diff
@@ -1,3 +1,4 @@
 default['optoro_memcached']['log_dir'] = '/var/log/memcached'
 default['optoro_memcached']['log_path'] = "#{node['optoro_memcached']['log_dir']}/memcached.log"
 default['optoro_memcached']['mem_size_mb'] = '512'
+default['monit']['address'] = 'localhost'
```

Comment:
Is this just the "from" base domain?  Or is it the complete address?  If it's the "from" can we make it the node[:fqdn] or something meaningful - Not a blocker.